### PR TITLE
I'd like to add support for SVG

### PR DESCRIPTION
**Pull request by washad**
If OK, I'd like to add support for SVG tags. There are a lot of tags, so I figured it better to submit a sample first. If you approve, I will subsequently add the remaining tags and tests for each. 

Regards, 
Steve J

### CODE REVIEW

File: dominate/tags.py [hunk 1]
```diff
@@ -754,6 +754,68 @@ class area(html_tag):
 
 
 
+# SVG
```
Comment (Knio) [inline dominate/tags.py]: I'd put this in a separate file/namespace and not with the rest of the HTML tags

File: tests/test_svg.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+from dominate.tags import svg, circle, rectangle
+import unittest
```
Comment (Knio) [inline tests/test_svg.py]: No need for `unittest` since we we're using pytest

**Comment by Knio** [comment]
Seems good. I left a couple review comments. I am not sure if more work will need to be done for supporting svg namespaces nicely in the output.

File: dominate/tags.py (hunk 1 above)
Comment (washad) [reply dominate/tags.py]: Moved svg tags to their own file -- as with the tests.

File: tests/test_svg.py (hunk 2 above)
Comment (washad) [reply tests/test_svg.py]: Switched to pytest.

**Comment by washad** [comment]
Should be at a point where you can look at it. One note, I added some support for converting underscore to dashed - it seems to be required in several instances for SVG. Not sure if the way I did it is the best, but my goal was not to touch your code. Feel free to re-write it or recommend changes.

**Comment by washad** [comment]
Sorry for so many commits - I'm not used to this backward compatibility thing so I'm figuring it out as I go along. Looks like I have all tests passing now.

**Comment by Knio** [comment]
Thanks! You can leave this PR open and I will review/merge it from here. I probably won't have time to look at it all until at least the weekend though.

**Comment by washad** [comment]
Think you will have a chance to look at this soon? I'd like to use the changes in a project I'm working on.

**Comment by Knio** [comment]
I just squash/rebased and cleaned a few things up, and merged this to master.
It's also been pushed to PyPI with version 2.4.0 ready to use.

Thanks for the PR!

**Pull request closed without merging**